cmd/main.go

```diff
@@ -28,8 +28,8 @@ import (
 // internal k8s service
 const (
 	AdminDiagnosticsPort     = 8877
-	DefaultSnapshotURLFmt    = "http://ambassador-admin:%d/snapshot-external"
-	DefaultDiagnosticsURLFmt = "http://ambassador-admin:%d/ambassador/v0/diag/?json=true"
+	DefaultSnapshotURLFmt    = "http://edge-stack-admin:%d/snapshot-external"
```

Comment:
This change is fine for Helm but breaks the default standalone yaml shipped with AES and Emissary-ingress (maybe). This needs to be coordinated with changes or probably a better long term approach would be to move away from Edgissary maintaining its own stuff and depend on this repository to own it.

Reply:
So since it is not necessary to fix the memory-leak can this be extracted into a separate PR and tested against AES helm and yaml?

Reply:
yep